jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/core/listeners/DslBackendListener.java

```diff
@@ -24,6 +24,11 @@
  */
 public abstract class DslBackendListener<T extends DslBackendListener<T>> extends BaseListener {
 
+  public static final String API_KEY_ARG = "apiKey";
+  public static final String API_URL_ARG = "datadogUrl";
```

Comment:
These variables are only used in datadog listener. Moving them here I think adds to confusion to code reviewers and adds unnecessary "dependency"/knowledge from parent class about the implementation of one particular subclass.